doc/en/doctest.rst

```diff
@@ -110,16 +122,7 @@ Python 3 unchanged:
 * ``ALLOW_BYTES``: when enabled, the ``b`` prefix is stripped from byte strings
   in expected doctest output.
 
-As with any other option flag, these flags can be enabled in ``pytest.ini`` using
```

Comment:
Where's the code for `ALLOW_UNICODE` and `ALLOW_BYTES`? Sybil has these for its doctest support, but I had to put some big caveats in because parsing `u'...'` and `b'...'` out of expected output when nested quotes are involved turns out to be really very hard.

Reply:
You can find it here: https://github.com/pytest-dev/pytest/blob/9c700d1fd5496c2f844f3dd20835ab204a822945/src/_pytest/doctest.py#L452-L504

Reply:
Cool, if I ever get desperate, I'll borrow those regexes, thanks! (but only a few more months of Py2, right?!)

Reply:
Yes, after that only critical bug fixes